src/components/App.tsx

```diff
@@ -33,8 +33,8 @@ export const App = () => {
 
   useEffect(() => {
     initializePlugin({pluginName: kPluginName, version: kVersion, dimensions: kInitialDimensions});
-    adjustStationDataset(weatherStations); //change max data to "present"
-    createStationsDataset(weatherStations); //send weather station data to CODAP
+    adjustStationDataset(weatherStations as IWeatherStation[]); //change max data to "present"
```

Comment:
Its fine for this PR but it would be good in a future PR to convert the imported json file to a `.ts` file and type the data in the file so that if the types change but the imported data does not it is caught at compile time.